mods/tuxemon/maps/professor_lab.tmx

```diff
@@ -53,13 +64,66 @@
     <property name="cond20" value="is player_facing down"/>
    </properties>
   </object>
-  <object id="69" name="Choose Creature 1" x="192" y="96" width="16" height="16"/>
-  <object id="74" name="Choose Creature 2" x="208" y="96" width="16" height="16"/>
-  <object id="75" name="Choose Creature 3" x="224" y="96" width="16" height="16"/>
+  <object id="69" name="Choose Rockitten" type="event" x="192.091" y="112.035" width="16" height="16">
+   <properties>
+    <property name="act10" value="add_monster rockitten,5"/>
+    <property name="act20" value="dialog you picked RockKitten"/>
```

Comment:
FYI, this is Rockitten, not RockKitten
https://wiki.tuxemon.org/index.php?title=Rockitten